tests/topotests/isis_srv6_te_topo1/test_isis_srv6_te_topo1.py

```diff
@@ -453,11 +456,208 @@ def test_srv6_te_policy_removed():
         !"
     )
 
-    for rname in ["rt1"]:
+    # for rname in ["rt1"]:
+    #    router_compare_json_output(
+    #        rname,
+    #        "show ipv6 route static json",
+    #        "step4/show_ipv6_route.ref",
+    #    )
+
+
+#
+# Step 5
+#
+# Test SRv6 End.B6.Encaps
+#
+
+
+def test_srv6_end_b6_encaps():
+    logger.info("Test (step 5): verify SRv6 END.B6.Encaps")
+    tgen = get_topogen()
+
+    # Skip if previous fatal error condition is raised
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    tgen.gears["rt1"].vtysh_cmd(
+        "configure \n \
+         ipv6 route fc00:0:6b::/48 fc00:0:6:: color 1 \n \
+		 segment-routing \n \
+		 traffic-eng \n \
+ 		 segment-list srv6 \n \
+		  index 1 ipv6-address fc00:0:2:: \n \
+		  index 2 ipv6-address fc00:0:6:: \n \
+		  exit \n \
+		  policy color 1 endpoint fc00:0:6:: \n \
+		   candidate-path preference 1 name srv6 explicit segment-list srv6 \n \
+		 exit \n \
+		exit \n \
+		!"
+    )
+
+    tgen.gears["rt2"].vtysh_cmd(
+        "configure \n \
+		 segment-routing \n \
+		 traffic-eng \n \
+		  segment-list srv6-header \n \
+		   index 1 ipv6-address fc00:0:4:: \n \
+		   index 2 ipv6-address fc00:0:5:: \n \
+		  exit \n \
+		  policy color 1 endpoint fc00:0:6:: \n \
+		   srv6-binding-sid fc00:0:2:: \n \
```

Comment:
The binding-sid value must not be the same as the End SID value.
Value has however to be taken from the same locator prefix configured in zebra.
Take for instance fc00:0:2:100::.

Reply:
To be sure we are talking about the same thing. You do not want to install the second route P "fc00:0:2::" that has higher priority (as of now, before changing the distance of p to 150)?

Please, explain why you don't want this change? 

SRv6 TE(pathd, or "p") instructs the packets in the network to follow the specified path instead of following the shortest path calculated by the IGP (I).


``
```
C>* 2001:db8:1::/64 is directly connected, eth-sw1, 00:00:17
I>* fc00:0:1::/48 [115/10] via fe80::38ef:20ff:fe7b:77a9, eth-sw1, weight 1, 00:00:13
p>* fc00:0:2::/128 [150/0] is directly connected, eth-rt4-2, seg6local End.B6.Encap nh6 fc00:0:2::, seg6 fc00:0:4::,fc00:0:5::, weight 1, 00:00:10
I   fc00:0:2::/128 [115/0] is directly connected, sr0 inactive, seg6local End USP, weight 1, 00:00:19

```